Formula/terraform-rover.rb

```diff
@@ -16,8 +16,8 @@ class TerraformRover < Formula
   end
 
   depends_on "go" => :build
+  depends_on "terraform" => :test
```

Comment:
Why?

Reply:
no need to install the latest `terraform`

Reply:
But you need terraform to use this tool. It is the literal definition of depending on terraform.

Reply:
yeah, but not the latest, it would potentially suffer the same issue as terragrunt and etc (tfstate compatibility issue or existing toolchain conflict).